README.md

```diff
@@ -210,6 +211,27 @@ For dex.decred.org, reach out
 [on Element](https://matrix.to/#/!mlRZqBtfWHrcmgdTWB:decred.org?via=decred.org&via=matrix.org&via=planetdecred.org)
 to appeal.
 
+## Fees
+
+The dex does not charge trading fees, but users have to pay on-chain transaction
+fees. Unlike on centralized exchanges where users can trade any arbitrary amount
+of funds, orders on the dex must be made in increments of a specified lot size.
```

Comment:
No reason to compare and contrast with centralized exchanges here. Just give them our reasoning.

> To ensure that on-chain transaction fees do not eat a significant portion of the order quantity, orders must be specified in increments of a minimum lot size. For example, if on-chain transaction fees worked out to $5, and a user was able to place an order to trade $10, they would lose half of their trade to transaction fees. So for high-fee assets, the operator must set a large lot size.